public/form_category.js

```diff
@@ -0,0 +1,43 @@
+// See https://symfony.com/doc/current/form/form_collections.html
```

Comment:
Are we using webpack in this project or not (cf. https://github.com/aakb/itstyr/blob/develop/webpack.config.js)?

Reply:
not